javascript-modal/main.js

```diff
@@ -0,0 +1,22 @@
+const $openButton = document.querySelector('.open-button');
+const $closeButton = document.querySelector('.close-button');
+const $overlay = document.querySelector('.overlay-light');
+const $modalContainer = document.querySelector('.modal-container-light');
+
+function openModal(event) {
+  $overlay.className = 'overlay-dark';
+  $modalContainer.className = 'modal-container-dark';
+  console.log('does this work 2');
```

Comment:
Remove all `console.log`s before submitting your code. They are only for testing / debugging purposes